library/src/main/java/com/mapzen/android/MapDataManager.java

```diff
@@ -0,0 +1,52 @@
+package com.mapzen.android;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Manages {@link PersistableMapData} objects for handling recreation of {@link MapData} objects
```

Comment:
Unfortunately Javadoc links require a fully qualified class name to resolve unless that class is in the same package or included via an `import` statement.

For example `{@link PersistableMapData}` resolves since it is in the same package but `{@link MapData}` does not and instead needs to be `{@link com.mapzen.tangram.MapData}`.